dataEng_container_tools/cla.py

```diff
@@ -361,7 +367,7 @@ def get_output_uris(self):
         
         Returns:
             A list of all output URIs passed in through the command line. URIs
-            are of the format 'gs://bucket_name/output_path/filename'.
+            are of the format 'gs://bucket_name/input_path/filename'.
```

Comment:
@haitiangk92 Should this be 'output_path' since  input_path is in the doc string already for get_input_uris() and line 370 refers to get_output_uris()? Unless I'm missing sosmthing which is probable

Reply:
yea, im just resolving merge conflicts with this PR. actual changes will follow

Reply:
Good will approve, looks good 👍